prepare_an4/prepare.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
Nice job with the file, I've created a prepare an4 script that does everything, from downloading to extracting. Code here is (here)[https://gist.github.com/SeanNaren/20608497249016ff645b942809d69f99]